### ISSUE TITLE
spatial_cnn overfitting

### ISSUE THREAD

**Issue by Robinxin123** [jeffreyhuang1/two-stream-action-recognition#57]
hello all,
when i run the spatial_cnn.py, i found the loss function for test is very strange. does someone have the same problem?  
many thanks!

/home/huxin/Bilder/overfitting.png

**Comment by Songyyan** [comment]
Hello, I am preparing to run this model. I don't know what this specific environmental requirement is. I want to ask you, thank you very much!

**Comment by xqqxiaoqiqi** [comment]
我也碰到了过拟合的问题，请问楼主后来解决了吗？

**Comment by Tariq195** [comment]
@Robinxin123 Can you tell me the learning rate you are keeping? 

I am getting these result, prec@1 and prec@2 on just two epochs but this is really so strange? 

![Capture4](https://user-images.githubusercontent.com/76560085/129671262-eb684ca0-cb88-4e26-906c-57397276ea4b.PNG)

**Comment by Tariq195** [comment]
Should I run all 500 epochs for training only? then check the validation or should I run both? because the epoch for validation is taking 21-22min and 19-20 mins for training, add them it will be 40mins, for each epoch it is taking 40 minutes. If I run it for 500 epochs it will take almost 13 days for training validation, I am using RTX 3080 10 GB + i9 10th generation.

**Comment by 202122209010** [comment]
您好，我遇到的问题是FileNotFoundError: [Errno 2] No such file or directory: 'dic/frame_count.pickle'，我该如何得到这个.pickle文件
